x-pack/filebeat/input/netflow/_meta/fields.yml

```diff
@@ -0,0 +1,1383 @@
+- key: netflow
+  title: "NetFlow"
+  description: >
+    Fields from NetFlow and IPFIX flows.
+  fields:
+    - name: netflow
+      type: group
+      description: >
+        Fields from NetFlow and IPFIX.
+      fields:
+        - name: type
+          type: keyword
+          description: >
+            The type of NetFlow record described by this event.
+
+        - name: exporter
+          type: group
+          description: >
+            Metadata related to the exporter device that generated this record.
+          fields:
+            - name: address
+              type: keyword
+              description: >
+                Exporter's network address in IP:port format.
+
+            - name: sourceId
+              type: long
+              description: >
+                Observation domain ID to which this record belongs.
+
+            - name: timestamp
+              type: date
+              description: >
+                Time and date of export.
+
+            - name: uptimeMillis
+              type: long
+              description: >
+                How long the exporter process has been running, in milliseconds.
+
+            - name: version
+              type: int
+              description: >
+                NetFlow version used.
+
+        - name: octetDeltaCount
```

Comment:
Perhaps it's worth a discussion if we want to automatically convert these to snake-case, so `octet_delta_count`. More consistent with the Beats and ECS, but more surprising to people that know NetFlow. @andrewkroh @MikePaquette  Do you have an opinion on this?

Reply:
Good question.  I vote for consistency with ECS naming rules over preserving the name that those familiar with the protocol might know, so `octet_delta_count`

Reply:
Thanks, now it's consistent with ECS. Unfortunately, that increases the PR size.